### ISSUE TITLE
Implement date ::= <relative_specifier> <unit>

### ISSUE THREAD

**Issue by Kxnr** [devinvs/fuzzydate#6]
While testing #5, I found that the `date ::= <relative_specifier> <unit>` construct is not supported as documented.

**Comment by colelawrence** [comment]
I found the same thing, so I implemented a mapping manually like:
```rs
        // manually fix some inputs
        let input = match input {
            "last week" => "1 week ago",
            "last month" => "1 month ago",
            "last year" => "1 year ago",
            "next week" => "1 week from now",
            "next month" => "1 month from now",
            "next year" => "1 year from now",
            _ => input,
        };
```

**Comment by Kxnr** [comment]
Could you test out the fix in #7? I think that fix there should fix this issue

**Comment by colelawrence** [comment]
Yes, that appeared to have fixed it for my branch. Thanks @Kxnr
